### DISCUSSION TITLE
Performance of data tables

### DISCUSSION THREAD

**Discussion by shah180** [orchidsoftware/platform#1576] [Q&A]
Hi, 

We have a table that loads around 5000 of rows of users. If I will edit any row (which goes to another screen) and comes back,  then every time, it sends a request to database to run a query. It puts extra performance burden on the server and also the performance of table loading is pretty slow as everytime, it waits for the query to execute

Is there any method or way to boost the performance? (for example using cache to store the result and only fetch from database when the filter or query changes?)

Any guidance or tips will be appreciated

Regards
Syed

**Comment by tabuna** [comment]
Hi, you can use the cache. You can read about it here https://laravel.com/docs/8.x/cache#introduction

**Comment by bilogic** [comment]
How about an edit button that opens in a new window?

**Comment by bilogic** [comment]
@shah180 can share some code of your data table? How about lazy loading?
